cms/templates/webinar_page.html

```diff
@@ -27,15 +27,18 @@ <h2>{{ page.sub_heading }}</h2>
   <div class="row">
     <div class="col-md-12">
       {% if page.date %}
-      <p>{{ page.formatted_date }} | {{ page.time }}</p>
+        <p>{{ page.formatted_date }} | {{ page.time }}</p>
       {% endif %}
-      {% if page.action_url and page.category|lower != 'upcoming' %}
+      {% if page.action_url and not page.is_upcoming_webinar %}
```

Comment:
Just curious.

1. Am I right to suspect that users can still visit upcoming webinars through a URL manually?
2. When there is no Action URL for an upcoming Webinar, the link takes the user to something like `http://xpro.odl.local:8053/webinars/None/`. I think we should handle this in the `serve` method of the Webinar's CMS page to return a 404 when an upcoming webinar is requested through a URL.
3. Once we achieve points 1 and 2 We will be in a position where the webinar detail page will only be served by on-demand webinars and we won't need the above `if/else` conditions. Just the action URL.

Reply:
**My Assumption**: I am assuming that the webinar detail page would only be served for on-demand webinars. If this is not true, My above comment might get invalidated.

Reply:
I actually suspected that we might need an upcoming webinar detail page in the future, so, that is the reason I kept the checks.